### DISCUSSION TITLE
Welcome to grail-ui Discussions!

### DISCUSSION THREAD

**Discussion by bekos** [grail-ui/grail-ui#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JVariance** [comment]
Hi @bekos ,
amazing work up to this point. Awesome to have such great utilities next to **accessible** components. 
Hope you keep working on this ♥
